secant/UIComponents/CircularFrame/CircularFrameBadge.swift

```diff
@@ -77,6 +77,7 @@ struct BadgesOverlay: Animatable, ViewModifier {
                                                 y: proxy.size.height * 0.15
                                             )
                                             .opacity(badgeIndex == viewStore.index ? 1 : 0)
+                                            .shadow(color: Asset.Colors.Onboarding.badgeShadow.color, radius: 10, x: 0, y: 0)
```

Comment:
[non-blocking | nit | styling] Tiny nitpick, this line is a bit long ( > 100 chars), we could split it as below (totally up to you / preference though 😄 )

```suggestion
                                            .shadow(
                                                color: Asset.Colors.Onboarding.badgeShadow.color,
                                                radius: 10,
                                                x: 0,
                                                y: 0
                                            )
```